client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -90,6 +90,39 @@
             />
           </b-form-group>
         </b-col>
+
+        <b-col
+          cols="12"
+          lg="6"
+        >
+          <b-form-group
+            :label="$t('record.horizontalEditMode')"
+            label-class="text-primary"
+          >
+            <c-input-checkbox
```

Comment:
I think instead of two checkboxes we'll have to have a dropdown
Not sure of the naming but it should be
- Default 
- All in one line
- Multiple in one line